src/langs/en.json

```diff
@@ -186,6 +186,7 @@
   "LIBRARY_SETTINGS_PUBLISH_NOTIFICATIONS_LABEL": "Send email notifications to all co-editors",
   "LIBRARY_SETTINGS_PUBLISHED_STATUS": "This element is published. Anyone can access it and is available on Graasp Library, our public repository of learning ressources.",
   "LIBRARY_SETTINGS_CHILD_PUBLISHED_STATUS": "This element is part of a collection that is already published. Publishing and un-publishing can be done at the collection root.",
+  "LIBRARY_SETTINGS_TYPE_NOT_ALLOWED_STATUS": "This element can not be published. Only folders are allowed to be published on Graasp Library.",
```

Comment:
Should we have dynamic text for types? I think we have translations for types as well in common.

Reply:
Yes it's an interesting point !